flink-python/docs/reference/pyflink.table/expressions.rst

```diff
@@ -195,6 +195,8 @@ string functions
     Expression.reverse
     Expression.split_index
     Expression.str_to_map
+    Expression.json_quote
+    Expression.json_unquote
```

Comment:
Shouldn't we put these functions here and in all other python related files together with other json functions?